### DISCUSSION TITLE
Set start position of a track in wavesurfer-multitrack programatically?

### DISCUSSION THREAD

**Discussion by kdaniek-tuney** [katspaugh/wavesurfer.js#3785] [Q&A]
Hey, thanks for an amazing library. I just wanted to ask if there's a way to change `startPosition` of specified track in the multitrack? We have a possibility to change the volume:

```
 multitrack.setTrackVolume(0, 1) // index, value
```

But is there a way to change more stuff of particular track? Like

```
multitrack.setStartPosition(0, 24)

// or

multitrack.setDraggable(0, false)
```

Thanks! :)

**Comment by katspaugh** [comment]
There’s no API for that (see the [docs](https://wavesurfer-multitrack.pages.dev/docs/classes/default)) but PRs are welcome. You can also destroy the whole multitrack and create a new one with a new startPosition for that track.

**Reply by kdaniek-tuney** [reply]
PR: https://github.com/katspaugh/wavesurfer-multitrack/pull/37
Please review, thanks:)
